### ISSUE TITLE
In user objects, do we need to store a Boolean value indicating if the user is banned or not?

### ISSUE THREAD

**Issue by Vim0315** [CSC207-UofT/course-project-codemonkeys#30]

**Comment by langsonzhang** [comment]
I say no. Since if a User is banned, he will be removed from the system.

**Comment by Vim0315** [comment]
I'm still concerned if we should remove the user from the system when it's banned. Removing a user from the system means that all the user's information will be deleted permanently. What if we need this information in the future?

**Comment by langsonzhang** [comment]
That is true. I think with our current authority system, we can simply give that User no authorities. This way we don't have to remove them from the system, but they can't do anything.
